app/components/support_interface/reasons_for_rejection_dashboard_component.html.erb

```diff
@@ -75,15 +75,6 @@
   sub_reasons_result: sub_reasons_for(:safeguarding_y_n),
 ) %>
 
-<%= render SupportInterface::ReasonForRejectionDashboardSectionComponent.new(
```

Comment:
It feels like any stats we show about this will be misleading, so I've removed it